.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,71 @@
+name: Build Docker Container
+
+on:
+  push:
+    tags:
+      - '*Release*'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/docker-release.yml'
+      - 'CMakeLists.txt'
+      - 'LICENSE'
+      - 'README.md'
```

Comment:
I do not think we should run the workflow for changes to license and readme file.

They barely ever change and only having them here for the cache is maybe more confusing than helpful. It should trigger on relevant changes so we test if the image still builds, but not only to keep the cache relevant.
I would much rather run generally on pushes to master (each merge) then.

Reply:
removed them!